diff --git a/tools/cythonize.py b/tools/cythonize.py
index 2963624..a64585d 100755
--- a/tools/cythonize.py
+++ b/tools/cythonize.py
@@ -29,7 +29,7 @@ def do(fn):
     # cythonize
     subprocess.Popen([cython, fn], env=os.environ).communicate()
 
-    if not package:
+    if not package or package == 'cymunk_python':
         print 'no need to rewrite', fn
     else:
         # get the .c, and change the initXXX
